cds_migrator_kit/rdm/migration/transform/xml_processing/rules/base.py

```diff
@@ -189,7 +194,10 @@ def subjects(self, key, value):
     def validate_subject_scheme(value, subfield):
         subject_scheme = value.get(subfield)
 
-        is_cern_scheme = subject_scheme and (
+        if not subject_scheme:
+            return True
+
+        is_cern_scheme = (
             subject_scheme.lower() == "szgecern" or subject_scheme.lower() == "cern"
         )
 
```

Comment:
can you simplify also the below condition?
```
is_author = subject_scheme.lower() == "author"
if not (is_cern_scheme or is_author):
    raise UnexpectedValue(field=key, subfield=subfield, value=subject_scheme)
```

Reply:
I rebased on top of your changes, I thought it was supposed to take precedence

Reply:
I mean your change now is much simpler so we could simplify what I have done below